django/contrib/admin/static/admin/css/base.css

```diff
@@ -239,7 +239,7 @@ code, pre {
 
 pre.literal-block {
     margin: 10px;
-    background: #eee;
+    background: var(--darkened-bg);
```

Comment:
Looks like that one was completely missed.

Are we happy that this will change from `#eeeeee` to `#f8f8f8` for the light theme? Can you provide a screenshot difference for the light also?

Reply:
Yes, I'm fine with this change, it's more consistent with other elements:

Before:
![image](https://user-images.githubusercontent.com/2865885/110334572-36691880-8023-11eb-9ed2-c10b68ba8516.png)
After:
![image](https://user-images.githubusercontent.com/2865885/110334689-526cba00-8023-11eb-8921-4bf55155284a.png)

Reply:
Also, in 0a802233ec1421e5e59a486be69daef9b112fd0d we've changed a lot of `#eee` to
- `#e8e8e8` (`hairline-color`), or
- `#e4e4e4` (`selected-bg`).

Reply:
Ok, sounds good. The CSS variables definitely help to bring consistency that was missing before.